starter-project-web-react/components/Blog/blogPostForm.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import {Box, TextField, Button} from '@mui/material'
+export default function BlogPostForm({
+    handleClose
+}) {
+  return (
+    <div>
```

Comment:
also if there is no purpose for having the div, it might just be fine to get rid of it and have the Box as the root component, because i don't see any purpose from my side now

Reply:
done